modules/social_features/social_group/src/Form/SocialGroupSettings.php

```diff
@@ -64,6 +128,48 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#weight' => 20,
     ];
 
+    // Cross-posting settings.
+    $form['cross_posting'] = [
+      '#type' => 'details',
+      '#title' => $this->t('Cross-posting settings'),
+      '#open' => FALSE,
+      '#weight' => 30,
+      '#tree' => TRUE,
+    ];
+
+    $form['cross_posting']['status'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Enable cross-posting'),
+      '#description' => $this->t('If enabled, one node can be added as a group content to multiple groups.'),
+      '#default_value' => $config->get('cross_posting.status'),
+    ];
+
+    $form['cross_posting']['content_types'] = [
+      '#type' => 'checkboxes',
+      '#title' => $this->t('Content types'),
+      '#options' => $this->getCrossPostingEntityTypesOptions(),
+      '#default_value' => $config->get('cross_posting.content_types') ?? [],
+      '#states' => [
+        'visible' => [
+          ':input[name="cross_posting[status]"]' => ['checked' => TRUE],
+        ],
+      ],
+    ];
+
+    // The group types list doesn't allowed to use in cross-posting.
+    $form['cross_posting']['group_types'] = [
+      '#type' => 'checkboxes',
+      '#title' => $this->t('Disable cross-posting for the group type:'),
```

Comment:
@nechai It's better to rename title and logic of this variable from `Disable` to `Enable`
Also to be consistent you can just rename title to the `Group types` and use description fo the explanation.